### PR TITLE
Added another chop question

### PR DESCRIPTION
**Pull request by LotemBY**
And modified question 2, so the questions won't be too similar.

### CODE REVIEW

File: docs/beginner/chop-question-4.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+id: chop-question-4
+title: The Chop (Question 4)
+---
+
+import BeginnersGuideProgress from '@site/src/beginnersGuide.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ChopQuestion3 from '@site/image-generator/yml/beginner/chop-question-4.yml';
```
Comment (DrAnax) [inline docs/beginner/chop-question-4.md:9]: Typo, change it to:

`import ChopQuestion4 from '@site/image-generator/yml/beginner/chop-question-4.yml';`

**Review by DrAnax — COMMENTED** [review]
There's a typo on docs/beginner/chop-question-4.md

**Comment by DrAnax** [comment]
Looks good now

**Comment by Zamiell** [comment]
thanks

**Pull request merged**